### DISCUSSION TITLE
GitHub Pages won't allow Https

### DISCUSSION THREAD

**Discussion by DodoSeal** [community/community#23440] [Other Feature Feedback, Questions, & Ideas]
<p>I recently setup a domain with my GitHub page but it won’t allow me to enforce Https. Even waiting weeks, it says it is not supported. My domain provider is Hostinger and I have checked all of my DNS records. I can go to the site and it works fine, just won’t let me use Https.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/e/4e24e93921562a83038cd2354e598550eeabbeea.png" data-download-href="/uploads/short-url/b9ik6fMWPm9zhzDBkB99LrZbNMK.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/53157397/181077105-66b87766-24ae-4eae-886d-7be2d739c00e.png" alt="image" data-base62-sha1="b9ik6fMWPm9zhzDBkB99LrZbNMK" width="690" height="216" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/e/4e24e93921562a83038cd2354e598550eeabbeea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">786×247 13.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by toomanylogins** [comment, marked as answer]
<p>Thats an apex domain. My guess is github cannot config the ssl cert as the apex points to hostinger. Try using a subdomain ie www…</p>

**Comment by mustofa-kamal** [comment]
I had the same issue. After adding www... , the problem solved.

thanks
mustofa
